### ISSUE TITLE
`bob use` breaks in 2.9.0

### ISSUE THREAD

**Issue by BSFishy** [MordechaiHadad/bob#198]
With the latest version of bob, `bob use` breaks.

```
~$ bob use stable
Apr 16 11:46:35.034  INFO Fetching latest version
Apr 16 11:46:35.866 ERROR Error: program not found
```

Environment:
 - OS: Windows 11 23H2
 - Installed from crates with `cargo install bob-nvim`

Steps:
 - `cargo install bob-nvim`
 - `bob use stable` (errors)
 - `cargo install bob-nvim@2.8.3`
 - `bob use stable` (succeeds)

**Comment by datwaft** [comment]
I also had the same problem earlier today in MacOS with bob installed from homebrew.

For some reason after building and installing Neovim from source it now works so it could have been an environment issue or could have been fixed because I downloaded neovim build dependencies.

**Comment by MordechaiHadad** [comment]
Hello apologies for the faulty release (did not take this edge case in consideration), could you test PR #199 ?

The following steps to follow in order to test:
1. Uninstall bob from your system
2. Reinstall the branch with: `cargo install --git https://github.com/MordechaiHadad/bob --branch fix/nvim-proxy`
3. Run `bob erase`
4. And then `bob use nightly`
5. And try to run neovim.

(copied this from issue #197)

**Comment by MordechaiHadad** [comment]
> > Owner
> 
> > The following steps to follow in order to test:
> > 
> > 1. Uninstall bob from your system
> > 2. Reinstall the branch with: `cargo install --git https://github.com/MordechaiHadad/bob --branch fix/nvim-proxy`
> > 3. Run `bob erase`
> > 4. And then `bob use nightly`
> > 5. And try to run neovim.
> 
> Yeah, it works :) Thanks @MordechaiHadad

I see thank you

**Comment by BSFishy** [comment]
Fixed in version 2.9.1
